gameofgame.py

```diff
@@ -86,15 +90,20 @@ def check_guess(guess, country):
         print("You both should learn about capital cities in the world!!!")
         
     # Print the current scores
-    print(f"Current scores: Player 1: {player1_score}, Player 2: {player2_score}")
+    print(
+        f"Current scores: Player 1: {player1_score}, Player 2: {player2_score}"
+        )
     
     # Ask if the players want to continue playing
     play_again = input("Do you want to play again? (y/n) ")
-    if play_again.lower() != "y":
+    if play_again.lower() != "n":
```

Comment:
Here, you have changed the code, so now if a player answers that they would like to play again (`y`), the game will end which is not the expected behaviour.